### PR TITLE
post 0.1.0 release items

### PR DESCRIPTION
**Pull request by nrbgt**
For #30

- [x] bumps to next version
- [x] adds release docs and `twine` for build
  - [x] removes conda-build (conda-forge still forthcoming)
  - [x] resolve build env
- [x] linting

### CODE REVIEW

File: examples/Tab_App_Example.ipynb [hunk 1]
```diff
@@ -1,130 +1,130 @@
...
-   "name": "python",
-   "nbconvert_exporter": "python",
-   "pygments_lexer": "ipython3",
-   "version": "3.7.8"
-  }
- },
- "nbformat": 4,
- "nbformat_minor": 4
-}
+{
```
Comment (zwelz3) [inline examples/Tab_App_Example.ipynb:1]: Is it STILL fudging with the line endings??

**Review by zwelz3 — APPROVED** [review]
~~Looks good. Is twine in support of future docs?~~ nvm looked it up on pypi

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -65,3 +65,17 @@ doit all
...
+## Releasing
+
+- After merging to `master`, download the `ipyradiant` dist artifacts after a successful
+  CI run
+- Check out `master`
+- Extract and inspect the files in `./dist`.
+- Ensure you have credentials for `pypi`
+
+```bash
+anaconda-project run --env-spec build twine upload dist/ipyradiant-*
```
Comment (nrbgt) [inline CONTRIBUTING.md:78]: twine is used to actually upload properly, with markdown support

File: examples/Tab_App_Example.ipynb (hunk 1 above)
Comment (nrbgt) [reply examples/Tab_App_Example.ipynb:1]: :shrug:

**Pull request merged**